packages/monitoring-experience-api/src/middleware/ipWhitelistValidation.ts

```diff
@@ -0,0 +1,43 @@
+import type { AbstractController, UnknownRequest } from '@verkkokauppa/core'
+import ipRangeCheck from 'ip-range-check'
+import { IpAddressValidationError } from '../errors'
+
+export const withIpWhitelistValidation = <
+  TController extends new (...args: any[]) => AbstractController
+>(
+  Controller: TController
+): TController => {
+  abstract class IpAddressWhitelistValidationController extends Controller {
+    protected async validateRequest(req: UnknownRequest) {
+      const validIps = process.env.ROUTE_PUBLIC_MONITORING_IP_WHITELIST
+        ? process.env.ROUTE_PUBLIC_MONITORING_IP_WHITELIST.split(' ')
```

Comment:
Could you add some development value to this environment variable, to files
.env.docker
.env.local.docker